### ISSUE TITLE
don't show link to "View all of README" when project page displays entire README 

### ISSUE THREAD

**Issue by skierpage** [isaacs/github#1011]
When visiting a project on GitHub you see an excerpt from its README file. This is always followed by a 
link to "View all of README" _even when_ the project page displays the entire README (because it's a small file). Example is https://github.com/redox-os/ransid . So you waste time following the link only to find no new information.

Suggested fix: on the project page, if it shows the entire README contents, replace the **View all of README** link with the text line "( ----- end of README file -----)".

**Comment by cirosantilli** [comment]
This is only on mobile I believe.

**Comment by skierpage** [comment]
@cirosantilli Yup, it seems recently the project page on desktop switched to including the entire README after the list of files. Maybe mobile will catch up.
